ros2_foxglove_bridge/src/message_definition_cache.cpp

```diff
@@ -15,6 +15,12 @@
 
 namespace foxglove {
 
+#ifdef _WIN32
+static const std::string SEPARATOR = "\\";
+#else
+static const std::string SEPARATOR = "/";
+#endif
+
```

Comment:
alternative: [std::filesystem::path::preferred_separator](https://en.cppreference.com/w/cpp/filesystem/path)

Reply:
I had that originally, but all of the #ifdefs required to include either experimental/filesystem or filesystem are a pain just to get this single character.